odo/backends/tests/test_bokeh.py

```diff
@@ -15,9 +15,8 @@
 
 def test_convert_dataframe_to_cds():
     cds = convert(ColumnDataSource, df)
-    assert cds.data == {'name': ['Alice', 'Bob', 'Charlie'],
-                        'balance': [100, 200, 300]}
-
+    assert list(cds.data['name']) == ['Alice', 'Bob', 'Charlie']
```

Comment:
This is orthogonal to the integral dtype change, correct?
(It's ok to leave in, especially since we want to  move towards getting tests passing. Just wanted to make sure that I wasn't missing some relation to the integral dtype detection.)

Reply:
Yes it prob should have be in a separate "Fix the tests" PR. I fixed 2 tests trying to get the CI green on this PR before I gave up. I'm happy to look into the other failures and see what I can do there to finish the job.

If I get around to it I'll make sure to open an issue first so people know I'm looking into it and we don't duplicate any work.